### ISSUE TITLE
Add Wrecking Ball

### ISSUE THREAD

**Issue by Harmon758** [Js41637/Overwatch-Item-Tracker#249]
https://playoverwatch.com/en-us/news/patch-notes/pc#patch-49154

**Comment by Js41637** [comment]
I'll be updating the site soon, sorry this has taken longer than it should. Blizzard have made some changes to the game that is delaying me being able to extract data from the game. We are looking into it.

**Comment by Js41637** [comment]
For the record, 1.26 stuff includes

 * Hammond/Wrecking Ball stuff
 * New Hanzo Simple Geometry voiceline
 * New OWL Grand Final rewards
 * Some item name renames
 * Something else I probably forgot.

**Comment by Js41637** [comment]
I pushed the hammond and other stuff, i haven't got previews of emotes/intros/sprays/icons/voicelines however as changes to the have broke how I can extract data and will take some time to figure it out.

The new OWL Sprays are there however and the lucio emote (no preview yet)

Skins and poses have previews though.

**Comment by matthompson** [comment]
You have a typo with one of the OWL Sprays. Watchpoint: Gibraltar should be Watchparty: Gibraltar

**Comment by Js41637** [comment]
This is fixed in my version and will be pushed soon, hopefully.

**Comment by Gonzom** [comment]
Any reason why you've named him "Hammond" and not "Wrecking Ball"? And if you keep that name, why place him in "W" instead of "H".

**Comment by Js41637** [comment]
Wrecking Ball is a stupid name also it's too long and requires 2 lines and I didn't want to make the text smaller.

**Comment by slowthgt** [comment]
Small detail, but the Spray "Watchpoint Gibraltar" from the OWL Finals is actually named "Watchparty: Gibraltar"

**Comment by Js41637** [comment]
Yeah i stuffed up the name, i've fixed it locally but i have a lot of other changes. I'll probably just hotfix it.

**Comment by Harmon758** [comment]
This was already pointed out and responded to a few comments ago 😐

**Comment by Js41637** [comment]
I just renamed the spray so it should be good now.

**Comment by slowthgt** [comment]
This is what I get for not reading the whole issue comments 😄
